COLLABORATOR_GUIDE.md

```diff
@@ -473,16 +473,20 @@ level.
 
 ### Involving the TSC
 
-Collaborators may opt to elevate pull requests or issues to the [TSC][] for
-discussion by assigning the `tsc-review` label or @-mentioning the
-`@nodejs/tsc` GitHub team. This should be done where a pull request:
+Collaborators may opt to elevate pull requests or issues to the [TSC][].
+This should be done where a pull request:
 
 - is labeled `semver-major`, or
 - has a significant impact on the codebase, or
 - is inherently controversial, or
 - has failed to reach consensus amongst the Collaborators who are
   actively participating in the discussion.
 
+Assign the `tsc-review` label or @-mention the
+`@nodejs/tsc` GitHub team if you want to elevate an issue to the [TSC][].
+Do not use the GitHub UI on the right hand side to assign to
+`@nodejs/tsc` or request a review from `@nodejs/tsc`.
```

Comment:
Would it still be fine to use it once in the beginning? That way it would show up for the whole TSC. Just assigning it a second time would be discouraged after the first TSC member looked at it?

Reply:
What's the hoped-for result that wouldn't be achieved with an @-mention?

I do think that getting TSC reviews is a bit of a problem right now. I think there are some culture/people changes that need to happen more than process changes to fix that, though.